Project/src/main/java/app/ArticleRepository.java

```diff
@@ -0,0 +1,10 @@
+package app;
+
+import app.models.Article;
+import app.models.User;
+import org.springframework.data.repository.CrudRepository;
+import java.util.List;
+
+public interface ArticleRepository extends CrudRepository<Article, Long> {
+    List<Article> findByUsers(User users);

```

Comment:
Since Article contains the "users" field in a ManyToMany relationship, this needs to be called "Users" otherwise Spring's reflection fails despite the fact that we're actually only getting by OneToMany in this specific case.